### PR TITLE
feat(wip): implement next-auth for expo

### PR DESCRIPTION
**Pull request by juliusmarminge**
**DRAFT THAT WILL BE UPSTREAMED TO AUTH.JS ONCE THEY HAVE MIGRATED**

Hot reload of the session is a bit meh still but this is a POC.

I also don't have an Apple Developer Subscription so I can't test the native provider.

Demo: https://cln.sh/G5CwgW0Z

### CODE REVIEW

**Comment by CuriosBasant** [comment]
What's the update on this?

**Comment by juliusmarminge** [comment]
> What's the update on this?

Waiting for @authjs/next

File: apps/expo/app.config.ts [hunk 1]
```diff
@@ -28,10 +52,14 @@ const defineConfig = (_ctx: ConfigContext): ExpoConfig => ({
   },
   extra: {
     eas: {
-      projectId: "your-project-id",
+      projectId: "768478b6-46cd-43a3-904b-f4d5065648d2",
```
Comment (ItsWendell) [inline apps/expo/app.config.ts:55]: This should be remove.d

**Comment by lmssiehdev** [comment]
will this be easier to implement with https://lucia-auth.com?

**Comment by dBianchii** [comment]
Hi, we're still waiting for @authjs/next?

**Comment by ARMATAV** [comment]
@juliusmarminge For this part you wrote;

>**DRAFT THAT WILL BE UPSTREAMED TO AUTH.JS ONCE THEY HAVE MIGRATED**

What are they migrating to exactly? v5?

**Pull request closed without merging**